data-visualize.qmd

```diff
@@ -83,7 +83,7 @@ A data frame is a rectangular collection of variables (in the columns) and obser
 
 Type the name of the data frame in the console and R will print a preview of its contents.
 Note that it says `tibble` on top of this preview.
-In the tidyverse, we use special data frames called tibbles that you will learn more about soon.
+In the tidyverse, we use special data frames called **tibbles** that you will learn more about soon.
```

Comment:
Do we talk more about tibbles?

Reply:
Yup, we do at https://r4ds.hadley.nz/data-transform.html#nycflights13.